postgrest/_async/client.py

```diff
@@ -76,24 +76,24 @@ def from_table(self, table: str) -> AsyncRequestBuilder:
         """Alias to :meth:`from_`."""
         return self.from_(table)
 
-    async def rpc(self, func: str, params: dict) -> AsyncFilterRequestBuilder:
+    async def rpc(self, func: str, params: dict) -> AsyncSelectRequestBuilder:
```

Comment:
is there any reason to use async for this function?
`await (await client.rpc(...)).execute()`

```suggestion
    def rpc(self, func: str, params: dict) -> AsyncSelectRequestBuilder:
```